projects/fsastorefrontlib/src/cms-components/checkout/components/add-options/add-options.component.spec.ts

```diff
@@ -191,4 +199,16 @@ describe('AddOptionsComponent', () => {
     component.ngOnInit();
     expect(component.currentCurrency).toEqual('EUR');
   });
+  it('should display classification features', () => {
+    component.ngOnInit();
```

Comment:
Is it necessary to call here `component.ngOnInit()` when you have it in `beforeEach`?